nikola/utils.py

```diff
@@ -104,11 +104,6 @@
 # Are you looking for 'generic_rss_renderer'?
 # It's defined in nikola.nikola.Nikola (the site object).
 
-# Aliases, previously for Python 2/3 compatibility.
```

Comment:
I don’t think we can remove this, plugins could still use those.

Reply:
I checked for those in getnikola/plugins.
Maybe this is a change that should be held off until a larger release.

Reply:
We can change everything now, add a standard `# TODO: remove in v9` deprecation notice to the codebase, and that will be taken care of in the future.

Reply:
I’ll handle this from here.